JS/modules/dateTime.js

```diff
@@ -0,0 +1,30 @@
+import { DateTime } from "./luxon.js";
+
+const dateLine = document.querySelector(".dayDate");
+const navBar = document.querySelector("nav");
+/* eslint-disable-next-line */
```

Comment:
- [ ] Please 🙏🏻 remove this `comment` because it is not required and also your code quality is also **compromising** here, I tried your project and was able to run and pass the **eslinter** test. Kindly remove this line and always use *eslinter* to disable `suggestions` when it is really either important or `mentioned` in your project requirement.